README.md

```diff
@@ -174,19 +174,13 @@ See the section about [running tests](https://facebook.github.io/create-react-ap
 
 Launches the test runner and generates coverage report
 
-### `yarn browser-test`
+### `test:e2e:install`
```

Comment:
Nitpicking: should include `yarn` before command. 👀

Reply:
Good catch, fixed ✅